front/migrations/20231204_author_backfill.ts

```diff
@@ -4,38 +4,62 @@ import {
   User,
   Workspace,
 } from "@app/lib/models";
+import { cons } from "fp-ts/lib/NonEmptyArray";
```

Comment:
At first, I thought you were going for a super fancy monad in the category of endofunctors but I realize this is likely VSCode auto-import doing its thing when you were trying console.log

Reply:
Oooh if only I could have used endofunfctors